numpy/core/_add_newdocs.py

```diff
@@ -5043,7 +5043,9 @@
 
     See Also
     --------
-    numpy.outer
+    numpy.outer : A less powerful version of ``np.multiply.outer`` that `ravel`\ s all inputs to 1D. This exists primarily for compatibility with old code.
+
+    tensordot : ``np.tensordot(a, b, axes=((), ()))`` and ``np.multiply.outer(a, b)`` behave same for all dimensions of a and b.
```

Comment:
If you use `\` in a docstring, you must make the docstring a raw string by prepending `r` before the start of the docstring: `r"""` instead of `"""`

Reply:
Or use `\\` in the docstring instead.